modules/ROOT/pages/to-describe-an-asset.adoc

```diff
@@ -49,6 +49,8 @@ In Markdown, use this syntax to reference a web served image:
 
 In Edit mode, you can reorder pages by clicking the page name and dragging the page up or down in the left nav bar.
 
+All page names are validated. Page names cannot start with `.`, cannot contain the characters `@`, `*`, `+`, `/`, `_`, or `\`, cannot be the reserved name `home`, cannot be the name of an existing page, and must be from 1 to 128 characters.
```

Comment:
I would do a bullet list just to make it cleaner, but it's a minor thing and you can take it or leave it. For example:
Page names must be from 1 to 128 characters in length. 
Page names cannot:
- Start with...
- Contain the characters...
- Use the reserved name `home`
- Be the name of an existing page

Reply:
Yes, I agree. Switched it to a bullet list. Thank you!